TLM/TLM/State/Options.cs

```diff
@@ -1,516 +1,466 @@
+            currentPanel.autoLayoutPadding.left = 10;
+            currentPanel.autoLayoutPadding.right = 10;
 
-			var parkAiGroup = panelHelper.AddGroup(Translation.GetString("Parking_AI"));
-			prohibitPocketCarsToggle = parkAiGroup.AddCheckbox(Translation.GetString("Enable_more_realistic_parking"), prohibitPocketCars, onProhibitPocketCarsChanged) as UICheckBox;
+            panelHelper = new UIHelper(currentPanel);
 
-			var ptGroup = panelHelper.AddGroup(Translation.GetString("Public_transport"));
-			realisticPublicTransportToggle = ptGroup.AddCheckbox(Translation.GetString("Prevent_excessive_transfers_at_public_transport_stations"), realisticPublicTransport, onRealisticPublicTransportChanged) as UICheckBox;
+            var keyboardGroup = panelHelper.AddGroup(Translation.GetString("Keybinds"));
+            ((UIPanel) ((UIHelper) keyboardGroup).self).gameObject.AddComponent<KeybindSettingsPage>();
+        }
 
-			// VEHICLE RESTRICTIONS
-			++tabIndex;
+        private static void MakeSettings_Maintenance(UITabstrip tabStrip, int tabIndex) {
+            UIPanel currentPanel;
+            UIHelper panelHelper;
+            AddOptionTab(tabStrip, Translation.GetString("Maintenance"));
+            tabStrip.selectedIndex = tabIndex;
 
-			AddOptionTab(tabStrip, Translation.GetString("Policies_&_Restrictions"));
-			tabStrip.selectedIndex = tabIndex;
+            currentPanel = tabStrip.tabContainer.components[tabIndex] as UIPanel;
+            currentPanel.autoLayout = true;
+            currentPanel.autoLayoutDirection = LayoutDirection.Vertical;
+            currentPanel.autoLayoutPadding.top = 5;
+            currentPanel.autoLayoutPadding.left = 10;
+            currentPanel.autoLayoutPadding.right = 10;
 
-			currentPanel = tabStrip.tabContainer.components[tabIndex] as UIPanel;
-			currentPanel.autoLayout = true;
-			currentPanel.autoLayoutDirection = LayoutDirection.Vertical;
-			currentPanel.autoLayoutPadding.top = 5;
-			currentPanel.autoLayoutPadding.left = 10;
-			currentPanel.autoLayoutPadding.right = 10;
+            panelHelper = new UIHelper(currentPanel);
 
-			panelHelper = new UIHelper(currentPanel);
+            var maintenanceGroup = panelHelper.AddGroup(Translation.GetString("Maintenance"));
 
-			var atJunctionsGroup = panelHelper.AddGroup(Translation.GetString("At_junctions"));
+            resetStuckEntitiesBtn = maintenanceGroup.AddButton(Translation.GetString("Reset_stuck_cims_and_vehicles"), onClickResetStuckEntities) as UIButton;
+            removeParkedVehiclesBtn = maintenanceGroup.AddButton(Translation.GetString("Remove_parked_vehicles"), onClickRemoveParkedVehicles) as UIButton;
 #if DEBUG
-			allRelaxedToggle = atJunctionsGroup.AddCheckbox(Translation.GetString("All_vehicles_may_ignore_lane_arrows"), allRelaxed, onAllRelaxedChanged) as UICheckBox;
+            resetSpeedLimitsBtn = maintenanceGroup.AddButton(Translation.GetString("Reset_custom_speed_limits"), onClickResetSpeedLimits) as UIButton;
```

Comment:
Is this debug feature something added by this PR or the earlier PR that added mph speeds?

Reply:
The reformatting and regrouping the code shifted lines, making diff really confusing.
I paid greater attention to not modify the code in here, just untabify and split into smaller functions.
So this was existing.